content/pt-br/service-discovery.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Descoberta de Serviço
+status: Concluído
+category: conceito
+tags: ["networking", "", ""]
+---
+
+## O que é
+
+A descoberta de serviço é o processo de localização de instâncias individuais que compõem um serviço.
+Uma ferramenta de descoberta de serviço rastreia os vários nós ou terminais que compõem um serviço.
+
+## Problema relacionado
+
+As arquiteturas nativas da nuvem são dinâmicas e fluidas, o que significa que estão em constante mudança.
+Um aplicativo [containerizado](/containerization/) provavelmente iniciará e interromperá várias vezes durante sua vida útil.
```

Comment:
I preffer "aplicação" instead of "aplicativo". If you like it, update text to maintain consistency. 

Minor: "contêinerização" instead of "containerizado"

Update link "/pt-br/containerization/"